uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/FeedScreen.kt

```diff
@@ -317,32 +322,29 @@ private fun FeedList(
                 items = feedContents.contents,
                 key = { _, item -> item.first.id }
             ) { index, (feedItem, favorited) ->
-                if (isHome && index == 0) {
+                if (isHome && index==0) {
                     if (isFilterState) {
                         FilterItemCountRow(feedContents.size.toString())
                     }
-                    FirstFeedItem(
-                        feedItem = feedItem,
-                        favorited = favorited,
-                        onClick = onClickFeed,
-                        showMediaLabel = isHome,
-                        onFavoriteChange = onFavoriteChange,
+                    DroidKaigi2021ArticleItem(
```

Comment:
The point I wrote in this issue is that I want to remove DroidKaigi2021ArticleItem in November. That's why I thought we will need to refactor. 💭 But we can create issue for that 👍 

Like this

```kotlin
object DroidKaigi2021 {
  fun isArticleEnd() = instant.isAfter(...)
}

val isDroidKaigiEnd = remember { mutableStateOf { DroidKaigi2021.isArticleEnd() } }

if (!isDroidKaigiEnd) {
    DroidKaigi2021ArticleItem(...)
} else {
...
}
```

https://github.com/DroidKaigi/conference-app-2021/issues/622
 `I'd like to show DroidKaigi2021 until November`

Reply:
Refactored at ca0bc3d
The logic is a bit complicated ...